core/chains/evm/logpoller/log_poller_internal_test.go

```diff
@@ -120,19 +121,19 @@ func TestLogPoller_RegisterFilter(t *testing.T) {
 	require.True(t, ok, "'Emitter Log 1 + 2 dupe' Filter missing")
 
 	// Removing an existing Filter should remove it from both memory and db
-	err = lp.UnregisterFilter("Emitter Log 1 + 2")
+	err = lp.UnregisterFilter(testutils.Context(t), "Emitter Log 1 + 2")
```

Comment:
```suggestion
	err = lp.UnregisterFilter(ctx, "Emitter Log 1 + 2")
```
Same for the ones below